hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAbfsNetworkStatistics.java

```diff
@@ -127,8 +128,10 @@ public void testAbfsHttpSendStatistics() throws IOException {
     // Operation: AbfsOutputStream close.
     // Network Stats calculation: 1 flush (with close) is send.
     // 1 flush request = 1 connection and 1 send request
-    expectedConnectionsMade++;
-    expectedRequestsSent++;
+    if (getPrefixMode(fs) == PrefixMode.DFS) {
```

Comment:
A comment briefly reasoning why it is not needed for blob could again be helpful.